.cci.jenkinsfile

```diff
@@ -68,7 +68,7 @@ cosaPod(runAsUser: 0, memory: "9Gi", cpu: "4") {
       coreos-assembler build
       coreos-assembler buildextend-metal
       coreos-assembler buildextend-metal4k
-      coreos-assembler buildextend-live --fast
+      coreos-assembler buildextend-live
```

Comment:
Note nowadays this can be written as

```
coreos-assembler osbuild metal metal4k live
```

So it's all done in a single invocation which is a bit more efficient.